backend/.well-known/assetlinks.json

```diff
@@ -0,0 +1,9 @@
+[{
+  "relation": ["delegate_permission/common.handle_all_urls"],
```

Comment:
It's not obvious what the `relation` properties listed provide us.  Since this is a JSON file, we don't have a good way of inlining documentation either.

Is there a way for us to document why the non-obvious properties have the value they do?  Naively, maybe we can document them in the integration test you added?

Reply:
changes done here https://github.com/Vonage/vonage-video-react-app/pull/202/commits/11d352836f7b69e9030af7f345dbc72e6dd28816